bench/abc/optimized/mapperCutUtils.ll

```diff
@@ -107,9 +107,8 @@ declare noundef i32 @printf(ptr noundef readonly captures(none), ...) local_unna
 ; Function Attrs: mustprogress nofree norecurse nosync nounwind willreturn memory(read, inaccessiblemem: none) uwtable
 define float @Map_CutGetRootArea(ptr noundef readonly captures(none) %0, i32 noundef %1) local_unnamed_addr #5 {
   %3 = sext i32 %1 to i64
-  %.idx = mul nsw i64 %3, 40
-  %4 = getelementptr i8, ptr %0, i64 96
-  %5 = getelementptr i8, ptr %4, i64 %.idx
+  %4 = getelementptr %struct.Map_MatchStruct_t_, ptr %0, i64 %3
+  %4 = getelementptr i8, ptr %4, i64 96
```

Comment:
We no longer trigger this ptradd canonicalization: https://github.com/llvm/llvm-project/blob/a4739708ebb99a7333658b96a0b328b6bfa0b193/llvm/lib/Transforms/InstCombine/InstructionCombining.cpp#L3018 This is not a problem (as the motivation for it is achieved more directly), but it introduces a lot of extra diffs.